Bottom: 56f111a2eb0510b0e47c288d8cd0008839b2267b
Top:    8af779957b3fd36a06a868e502807df2b67a28d2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-27 11:20:41 +0100

Refresh of try-to-make-gdk-and-gl-version

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 851f847..b801808 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -41,6 +41,7 @@ static hidGC current_gc = NULL;
 
 static int cur_mask = -1;
 
+static void ghid_draw_grid ();
 
 struct render_priv {
   GdkGLConfig *glconfig;
